packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -369,7 +377,10 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
         }
 
         this._baseDmmf = new BaseDMMFHelper(config.document)
-        const engineProtocol = getQueryEngineProtocol(config.generator)
+        const engineProtocol = NODE_CLIENT
```

Comment:
Could we pass `config` to `getQueryEngineProtocol` and move this logic in there along with the other logic?

Reply:
(In theory, NODE_CLIENT should be able to work there too, I think)

Reply:
`getQueryEngineProtocol` lives in `@prisma/internals` and is not build with `NODE_CLIENT` defined.

Reply:
`getQueryEngineProtocol` lives in `@prisma/internals`. Wouldn't using `NODE_CLIENT` there mean we'd have to build a separate edge version of internals?

Reply:
Somehow I thought that esbuild would inject across the whole bundle, but that does not work.

Reply:
Otherwise, I guess you could pass the `NODE_CLIENT` value as a param